lib/query.js

```diff
@@ -104,7 +104,13 @@ class Query {
 		}
 
 		if (Array.isArray(field)) {
-			field.forEach(field => this.sort(field));
+			// The array syntax `.sort([['field', 1], ['test', -1]])` can only be used with [mongodb driver >= 2.0.46](https://github.com/mongodb/node-mongodb-native/blob/2.1/HISTORY.md#2046-2015-10-15).
+
+			if (field.length > 0 && Array.isArray(field[0])) {
```

Comment:
I think it should be the other way around, array check first and then `length > 0`.